packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -160,13 +166,29 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
       <Stack className={chatArea} tokens={participantListContainerPadding} horizontal grow>
         <Stack className={chatWrapper} grow>
           {options?.errorBar !== false && <ErrorBar {...errorBarProps} />}
+          {
+            /* @conditional-compile-remove(file-sharing) */
+            <FileDownloadErrorBar
+              onDismissDownloadErrorMessage={() => {
```

Comment:
Memoize this closure with `useCallback` to avoid rendering `FileDownloadErrorBar` on each render pass.

Reply:
done